launch/localize_mapservice.launch

```diff
@@ -27,6 +27,12 @@
 		publish_odom: whether or not to publish inferred pose as an odometry message.
 		              publishes on /pf/pose/odom
 	 -->
+	<!-- Launch the initial pose publisher node -->
+	<node pkg="particle_filter" type="initialpose_publisher.py" name="initialpose_publisher" output="screen"/>
+
+	<!-- Launch the customized map service node-->
+	<node pkg="map_service" type="revit_map_test" name="revit_map_test"/>
```

Comment:
**Please avoid writing launch files that directly depends on `map_service` in this rospkg**, as this rospkg is meant to run onboard and `map_service` is meant to run on central server (handles multi-agent map merging as well in future). Normally they will be placed in different ROS workspaces. 

Writing a launch file that runs particle filter without any map is sufficient. We can manually run the map server in another terminal.